user_docs/en/changes.t2t

```diff
@@ -8,9 +8,9 @@
 == New Features ==
 - A Braille setting has been added to "show messages indefinitely". (#6669)
 - In Microsoft Outlook message lists, report if a message is flagged. (#6374)
-- In Microsoft Powerpoint, the exact type of a shape is now reported when editing a slide (Examples include: triangle, circle, video, arrow), rather than just 'shape'. (#7111)
+- In Microsoft PowerPoint, the exact type of a shape is now reported when editing a slide (Examples include: triangle, circle, video, arrow), rather than just 'shape'. (#7111)
 - Mathematical content (provided as MathML) is now supported in Google Chrome. (#7184)
-- NVDA can now speak using the new Windows OneCore voices (also known as mobile voices) included in Windows 10. You access these by selecting Windows OneCore voices in NVDA's Synthesizer dialog. (#6159)
+- NVDA can now speak using the new Windows OneCore voices (also known as SAPI5 Mobile voices) included in Windows 10. You access these by selecting Windows OneCore voices in NVDA's Synthesizer dialog. (#6159)

```

Comment:
No one calls these SAPI5 Mobile apart from an old NVDA driver. Also, the voices were previously named "Mobile"; e.g. "Microsoft Zira Mobile". I don't think SAPI5 should be mentioned here.